data.sql

```diff
@@ -1,5 +1,14 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+    VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
+	
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+    VALUES ('Gabumon', '2015-11-15', 2, true, 8);
```

Comment:
Great job @Mikey1-nuel ✋

- [ ] Kindly ensure you entered the correct `date_of_birth` for `Gabumon` as it is given in the project requirements.